db/schema.rb

```diff
@@ -638,6 +638,7 @@
     t.boolean  "exporting_photos",                                 default: false
     t.string   "color_theme",                        limit: 255
     t.boolean  "post_default_public",                              default: false
+    t.boolean  "chat_auto_login",                                  default: false
```

Comment:
This PR is about "auto-login on every sign-in" and I agree with @goobertron here, that shouldn't be the default on.

There is no "remember my last chat-status and restore that on next sign-in"-feature yet. I think that could be useful, but I should also be "offline" on the first use.